services/api-server/src/simcore_service_api_server/models/schemas/model_adapter.py

```diff
@@ -121,3 +125,20 @@ class ServicePricingPlanGetLegacy(BaseModel):
 assert set(ServicePricingPlanGetLegacy.model_fields.keys()) == set(
     _ServicePricingPlanGet.model_fields.keys()
 )
+
+
+class LicensedItemGet(BaseModel):
```

Comment:
Cool, thanks! Can we somehow hide this functionality and make it available only for Manuel? I guess this will still evolve a bit, with a few potential breaking changes.

Reply:
Yep, good point. I will exclude them for now. That has the disadvantage that backwards compatibility breaking changes will not be caught by our test. But we can add them once they are more stable.